src/core.js

```diff
@@ -11,7 +11,11 @@ import { defined, extend, objectType, is, now } from "./core/utilities";
 import { registerLoggingCallbacks, runLoggingCallbacks } from "./core/logging";
 import { sourceFromStacktrace } from "./core/stacktrace";
 
-const QUnit = {};
+var QUnit = {};
```

Comment:
You are right, it was just my IDE fooling me, i don't need it - will revert it to const thanks!